app/models/page_view.rb

```diff
@@ -6,7 +6,7 @@ class PageView < ApplicationRecord
 
   before_create :extract_domain_and_path
 
-  algoliasearch index_name: "UserHistory", per_environment: true, if: :belongs_to_pro_user? do
+  algoliasearch index_name: "UserHistory", per_environment: true, if: :belongs_to_pro_user?, enqueue: :trigger_index_sync do
```

Comment:
I noticed that we have inconsistent (but similar) naming on the enqueue methods. Most of them are `trigger_index`, but `User` model has `trigger_delayed_index`. What do you think about using `trigger_index` as a name for consistency?

Reply:
I like sync bc sometimes we are removing a record and not actually indexing it. I also took the delayed part out to avoid any delayed_job confusion that it might cause. What do you think about keeping sync?

Reply:
I'm not against `sync` part, the inconsistency bothers me a bit. In my opinion, "delayed" here is not about delayed job but delayed process, but I agree that we shouldn't add more confusion because "delayed" is kind of ambiguous.
Anyway, I don't think that inconsistent naming is a big deal here.